websites/R/Reddit/presence.ts

```diff
@@ -40,7 +40,7 @@ const presences: { [key in PresenceClients]?: Presence } = {
 		[PresenceClients.Reddit]: presence,
 	},
 	startTimestamp = Math.floor(Date.now() / 1000),
-	oldReddit = document.querySelector("._1tvdSTbdxaK-BnUbzUIqIY") === null;
+	oldReddit = document.querySelector("#default-header") === null;
```

Comment:
I'm unable to find this element on my new or old reddit.

Reply:
Interesting
![image](https://github.com/PreMiD/Presences/assets/42322979/82dfc886-dde6-4811-8585-7881b8028de3)